src/main/java/org/jenkinsci/plugins/envinject/EnvInjectComputerListener.java

```diff
@@ -53,10 +55,13 @@ private EnvVars getNewMasterEnvironmentVariables(Computer c, FilePath nodePath,
                 globalPropertiesEnvVars.putAll(((EnvironmentVariablesNodeProperty) nodeProperty).getEnvVars());
             }
 
-            if (nodeProperty instanceof EnvInjectNodeProperty) {
+            final Node node = c.getNode();
```

Comment:
Yes. It prevents occasional modifications in the code (e.g. <code>=</code> instead of <code>==</code>)